ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getAccountLink } from '@metamask/etherscan-link';
+import {
+  getCurrentChainId,
+  getRpcPrefsForCurrentProvider,
+} from '../../../../../../selectors';
+import { I18nContext } from '../../../../../../contexts/i18n';
+
+import Button from '../../../../../ui/button';
+import Tooltip from '../../../../../ui/tooltip';
+
+const Accreditation = ({ fetchVia, address }) => {
+  const t = useContext(I18nContext);
+  const chainId = useSelector(getCurrentChainId);
+  const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
+  const addressLink = getAccountLink(address, chainId, rpcPrefs);
+
+  const AccreditationLink = () => {
+    return (
+      <>
+        <div className="accreditation__prefix">
+          {t('transactionDecodingAccreditationVerified')}
+        </div>
+        <div className="accreditation__content">
+          <Button
+            type="link"
+            className="accreditation__content--link"
+            onClick={() => {
+              global.platform.openTab({
+                url: addressLink,
+              });
+            }}
+            target="_blank"
+            rel="noopener noreferrer"
+            title={t('etherscanView')}
+          >
+            {fetchVia}
+          </Button>
+        </div>
+        <div className="accreditation__suffix">
+          {t('transactionDecodingAccreditationDecoded')}
+        </div>
```

Comment:
I would suggest using the design system components here
```
import { TYPOGRAPHY } from '../../../../../../helpers/constants/design-system';
import Typography from '../../../../../ui/typography';
```

```suggestion
        <Typography
          variant={TYPOGRAPHY.H7}
          className="accreditation__prefix"
          boxProps={{ margin: 0 }}
        >
          {t('transactionDecodingAccreditationVerified')}
        </Typography>
        <Button
          type="link"
          className="accreditation__link"
          onClick={() => {
            global.platform.openTab({
              url: addressLink,
            });
          }}
          target="_blank"
          rel="noopener noreferrer"
          title={t('etherscanView')}
        >
          {fetchVia}
        </Button>
        <Typography variant={TYPOGRAPHY.H7} boxProps={{ margin: 0 }}>
          {t('transactionDecodingAccreditationDecoded')}
        </Typography>
```